bundles/org.aposin.gem.ui/src/org/aposin/gem/ui/dialog/progress/internal/CliProgressMonitorDialog.java

```diff
@@ -147,8 +157,52 @@ private void createCommandDetailsButton(final Composite parent) {
         commandDetailsButton = new Button(composite, SWT.TOGGLE);
         commandDetailsButton.setText(IDialogConstants.SHOW_DETAILS_LABEL);
         // create cursor
-        final Cursor buttonCursor = new Cursor(commandDetailsButton.getDisplay(), SWT.CURSOR_ARROW);
-        commandDetailsButton.setCursor(buttonCursor);
+        setArrowCursor(commandDetailsButton);
+
+    }
+
+    private void setArrowCursor(final Control control) {
+        final Cursor buttonCursor = new Cursor(control.getDisplay(), SWT.CURSOR_ARROW);
+        control.setCursor(buttonCursor);
+        controlsWithCursors.put(control, buttonCursor);
+    }
+
+    private Composite createCommandTabFolder(final Composite parent) {
+        // TODO: if only one command, do we really need the tab-folder?
+        final CTabFolder tabFolder = new CTabFolder(parent, SWT.TOP);

```

Comment:
I think, keeping a single tab-folder is better than changing the layout and interface (consistency). Furthermore, the recognition value is higher for the user.

Reply:
Leftover for some considerations I have when I implemented. Next commit will remove this and other TODOs that are not any longer relevant.